react/global.css

```diff
@@ -1,3 +1,47 @@
-.vtex-impersonate-customer {
+.vtex-telemarketing {
+  text-align: center;
+}
+
+.vtex-telemarketing .vtex-popover__arrow-up {
+  background-color: rgba(0, 0, 0, 0.9);
+  right: 20px;
+}
+
+.vtex-telemarketing .vtex-popover__box {
+  top: 3rem;
+  min-width: 320px;
+}
+
+.vtex-telemarketing__login-form-message {
   text-align: left;
 }
+
+.vtex-telemarketing__popover-header-email {
+  color: rgba(255, 255, 255, 0.5);
+}
+
+/* Logout */
+
+.vtex-telemarketing .vtex-telemarketing__logout .vtex-popover__arrow-up {
+  background-color: #ff4c4c;
+}
+
+/* Popover css */
+
+.vtex-popover__arrow-up {
+  transform: rotate(45deg);
+  width: 25px;
+  height: 25px;
+  right: 7px;
+  z-index: 2147483646;
```

Comment:
`z-max`?

Reply:
It is not the z-max because this is the arrow and it needs to be behind the box, which is the actual z-max

Reply:
ok

Reply:
could be `z-999` and the arrow `z-max`